### ISSUE TITLE
nn.functional.sigmoid is deprecated

### ISSUE THREAD

**Issue by KutisBayag** [alievk/avatarify#136]
please help, both screen appears but there's too much lag, like 1 frame per 15sec.

F:\Deepfake\conda\envs\avatarify\lib\site-packages\torch\nn\functional.py:2423: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
F:\Deepfake\conda\envs\avatarify\lib\site-packages\torch\nn\functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

**Comment by alievk** [comment]
Closing since problem description is insufficient.
